app.rb

```diff
@@ -61,12 +61,19 @@ class MakersBNB < Sinatra::Base
 
   get '/spaces/bookings/:id' do
     id = params[:id]
+    p id
+    session[:id] = id
     @record = Space.first(:id => id)
     erb :'spaces/bookings'
   end
 
-  # get '/spaces/requests/:id' do
-  #   erb :'spaces/requests'
-  # end
+  post '/spaces/request' do
+    space_id = session[:id]
+    # flash messsage 'Request to book has been sent'
+    booking = Booking.create(arrival: params[:request_from], departure: params[:request_until], space_id: space_id)
+    p booking
+    p booking.saved?
```

Comment:
Do these still need to print at the commit stage?

Reply:
see above